### ISSUE TITLE
test_analytical_correlation_between_trees output difficult to interpret

### ISSUE THREAD

**Issue by jeromekelleher** [tskit-dev/msprime#1092]
The output of the [test_analytical_correlation_between_trees](https://github.com/tskit-dev/msprime/blob/master/verification.py#L2431) verification test is difficult to interpret. I can't really tell what's being plotted and whether the results are correct.

@TPPSellinger, I think you wrote this? Would you mind having a look over it again, and see if the output can be made a bit more obvious to someone scanning hundreds of qqplots before a release of msprime?

**Comment by TPPSellinger** [comment]
I don't remember writing this but I can go through it anyway and let you know what it does.

**Comment by jeromekelleher** [comment]
Thanks @TPPSellinger, that would be really helpful! Ah, I see there's some gc stuff in there - maybe it was @fbaumdicker ?

(I should be able to use git blame, I know, but this has gone through a mega refactor and I can't be bothered doing the git archaeology.)

**Comment by fbaumdicker** [comment]
Yes, this is a part I have added. It plots the probability that two trees have the same MRCA (y-axis) along the chromosome (x-axis). The dashed line is the expected fraction of trees with equal MRCA, according to the formula from the Wiuf and Hein paper. The solid line is the simulated fraction of equal trees. The trees are compared to the tree at the first, last, and middle position, for three sets of gc parameters.

**Comment by jeromekelleher** [comment]
Thanks @fbaumdicker - would you mind updating it a little to add some axis labels and so on? Only if you have time - I understand you've got a lot on your plate.

**Comment by fbaumdicker** [comment]
I will try to have a look at it this week. When I update the figures I will notify you here again.

**Comment by fbaumdicker** [comment]
I updated the figures with labels and a legend. However, by doing so I noticed that the verification results no longer look like they should look. Currently, the plot looks like:

![prob_first](https://user-images.githubusercontent.com/13665835/87204049-2e29d380-c304-11ea-851a-54d9da470ac3.png)

But it should look like this:

![prob_first](https://user-images.githubusercontent.com/13665835/87204095-48fc4800-c304-11ea-99e3-817800755b7a.png)


The difference seems to be a scaling issue with Ne or so introduced in c0dc11c61a1a632b0c5b4e92e0c78bcf47f2ec2d

@jeromekelleher,  what has changed in this commit? I hope I simply need to adjust the parameters?
I will try to fix the scaling problem within this issue, but it would be good to get your input on this.

**Comment by jeromekelleher** [comment]
Aha, no wonder I couldn't read the output! Resolving the internal scaling issues was a long drawn out saga @fbaumdicker; basically, it turns out that I had massively complicated things for the sake of multiplying and then dividing times by the same number. The old "reference size" concept actually did nothing.

However, it shouldn't have affected these results. I think the best thing to do is to see what changes you need to make to get the figures working again, and we'll see if these make sense now. 

Thanks for following up on this.

**Comment by fbaumdicker** [comment]
I had to divide the gene conversion rate used in the simulation for the verification by 4. In addition, I added axis labels and a legend to the plots. The current implementation uses 1.000 reps, which suffices to see major deviations (like above) and is quite fast. If we use 100.000 reps the fits get almost perfect but the running time is correspondingly higher. I think 1.000 reps should be fine for the verification.py file.
